test/units/modules/network/check_point/test_cp_mgmt_network.py

```diff
@@ -46,13 +46,6 @@
 class TestCheckpointNetwork(object):
```

Comment:
on this tests, even if not affected by this PR, please change it in order not to get this unused import.
`test/units/modules/network/check_point/test_cp_mgmt_network.py:22:1: F401 'units.modules.utils.AnsibleFailJson' imported but unused
from units.modules.utils import set_module_args, exit_json, fail_json, AnsibleFailJson, AnsibleExitJson`

Reply:
I will raise another PR for `test_cp_mgmt_network`, in order to avoid conflicts I didn't raised that PR yet..

Reply:
but I have `test_cp_mgmt_network` ready for commit..

Reply:
there's no need for another PR, you can just add another commit to this one changing that ;)

Reply:
but I think it would create conflicts for the other 3 PRs, wouldn't it?

Reply:
well, you'd have to rebase your another PRs and sort out the issues, ofc. I'd rather do that but if you prefer to handle it then go ahead.